PlayerController/playercore.js

```diff
@@ -164,7 +164,7 @@ var gameName = document.title;
 
 function setGameName(text) {
     $("#gameTitle").remove();
-    document.title = text;
+    document.title = s;

```

Comment:
Missed a bit! This should be reverted.
```suggestion
    document.title = text;
```

Reply:
Fixed. (Sorry about that. I usually run a game after making changes like this.)